pkg/agent/config.go

```diff
@@ -136,4 +136,6 @@ type Config struct {
 	KafkaSASLClientSecretPath string `env:"KAFKA_SASL_CLIENT_SECRET_PATH"`
 	// ProfilePort sets the listening port for Go's Pprof tool. If it is not set, profile is disabled
 	ProfilePort int `env:"PROFILE_PORT"`
+	// GoMemLimit sets soft memory cap to ebpf agent process
+	GoMemLimit string `env:"GOMEM_LIMIT"`
```

Comment:
Why not using directly the GOMEMLIMIT env? That would avoid having to sync the env and the config. You would have it directly set by the operator.

Reply:
I thought we wanted to control this value from operator ?

Reply:
I get it operator will set the envvar nothing needed here I will remove this piece